.ci/install/isal.sh

```diff
@@ -2,15 +2,22 @@
 
 set -ex
 
-ISAL_MIRROR=http://kevinlin.web.rice.edu/static/isal.tar.gz
+ISAL_MIRROR=http://kevinlin.web.rice.edu/static/isal-2.tar.gz
 
 cd $HOME
-if [ ! -d /tmp/cache/isal ]; then
-    wget --quiet -O isal.tar.gz $ISAL_MIRROR
-    tar -xf isal.tar.gz -C /tmp
-    mv /tmp/isa-l_open_src_2.13 /tmp/cache/isal
+if [ -d /tmp/cache/isal ]; then
     cd /tmp/cache/isal
-    make
+    make clean
     cd $HOME
+    rm -rf /tmp/cache/isal
 fi
+wget --quiet -O isal.tar.gz $ISAL_MIRROR
+tar -xf isal.tar.gz -C /tmp
+mv /tmp/isal /tmp/cache/isal
```

Comment:
Following up to the above comment: you should probably just extract the tarball into $HOME, cd and build in that directory, and remove the last line of the script that symlinks it to `/tmp/cache/isal`.